### PR TITLE
Nav bar private pages dev

### LINKED ISSUE #27: As a developer, I need to build a navigation bar for each page.

### LINKED ISSUE #31: As a developer we need Public and Private Layouts for the app

### LINKED ISSUE #51: As a developer, I need to install react router and configure the routes.

### PR DESCRIPTION
**Pull request by XeniyaDob**
## Description

Responsive functional navbar. Layouts and routes implemented for the app.
## Resouce links or research articles

https://mui.com/material-ui/getting-started/installation/
npm install @mui/icons-material
https://reactrouter.com/en/v6.3.0/getting-started/installation#basic-installation
npm install react-router-dom@6
## Related Issue
closes #27 
closes #31 
closes #51 

## Acceptance Criteria
The functional navbar is created, and each button leads to the corresponding page.
Public and Private Layouts created
Public layout for the landing page, sign/signup page, and logout.
Private Layout with Navbar for all the rest pages.
React router installed. Routes configured.

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

### After
Screen Shot 2022-10-31 at 12 33 27 PM

Screen Shot 2022-10-31 at 12 32 18 PM
Screen Shot 2022-10-31 at 12 32 08 PM

### CODE REVIEW

File: src/components/Navbar/ResponsiveAppBar.js [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+import Toolbar from "@mui/material/Toolbar";
+import IconButton from "@mui/material/IconButton";
+import Typography from "@mui/material/Typography";
+import Menu from "@mui/material/Menu";
+import MenuIcon from "@mui/icons-material/Menu";
+import Container from "@mui/material/Container";
+import Avatar from "@mui/material/Avatar";
+import Button from "@mui/material/Button";
+import Tooltip from "@mui/material/Tooltip";
+import MenuItem from "@mui/material/MenuItem";
```
Comment (elenamagay) [inline src/components/Navbar/ResponsiveAppBar.js:13]: You can combine them all together in one line since they are from mui/material. Thanks

**Review by elenamagay — APPROVED** [review]
Great job Xeniya!

**Review by nuriyealp — APPROVED** [review]
looks good

File: src/components/Navbar/ResponsiveAppBar.js (hunk 1 above)
Comment (XeniyaDob) [reply src/components/Navbar/ResponsiveAppBar.js:13]: As discussed in the meeting, we need to keep them separately.

**Pull request merged**